math/src/algebra.rs

```diff
@@ -200,7 +201,7 @@ pub trait Space<R>:
     /// that they have the same length.
     ///
     /// For empty slices, the result should be [`Additive::zero`];
-    fn msm(points: &[Self], scalars: &[R], _concurrency: usize) -> Self {
+    fn msm<S: ParStrategy>(points: &[Self], scalars: &[R], _strategy: &S) -> Self {
```

Comment:
nit: can we just use `Strategy` in places like this? I tend to only rename if there is an import conflict?